app/javascript/onboarding/components/FollowTags.jsx

```diff
@@ -90,7 +90,9 @@ class FollowTags extends Component {
     }
 
     const classStyle =
```

Comment:
The changes in this file made by @ludwiczakpawel are not directly related to the profile onboarding but these are small changes for the UI design on the modal to try and keep the modal at the same height so that it doesn't look so jarring when it adjusts the height based on the content.